project_task_id_in_display_name/README.rst

```diff
@@ -0,0 +1,12 @@
+Project Task ID In Display Name
+===============================
+This module displays the ID of a task in its displayed name.
+
+Searching a task by its ID
+--------------------------
+When searching a task in kanban view, you may enter its ID instead of its name.
+The system will return the task if the entered expression matches its ID.
+
```

Comment:
Hello @rivo2302 
We need to enrich the Readme by adding some pictures and the feature of search in portal by task ID